Gruntfile.js

```diff
@@ -159,11 +159,11 @@ module.exports = function( grunt ) {
 				options: {
 					preserveComments: false,
 					sourceMap: true,
-					ASCIIOnly: true,
+					ie8: true,
```

Comment:
Could you add a comment explaining why that's needed with a link to the UglifyJS bug (and maybe a support comment for IE<11)?